recipes/cookbooks/mariadb/recipes/install_community.rb

```diff
@@ -3,6 +3,11 @@
 include_recipe "mariadb::mdbcrepos"
 
 
+# install net-tools
```

Comment:
It is my mistake, I dont create branch for this fix and I decided to put it in my current branch (6576).